apps/src/client.rs

```diff
@@ -378,38 +410,113 @@ pub fn connect(
             si_conn.handle_quack_acks(&mut conn, &mut buf, &app_data_start);
         }
 
-        // Generate outgoing QUIC packets and send them on the UDP socket, until
-        // quiche reports that there are no more packets to be sent.
-        loop {
-            let (write, send_info) = match conn.send(&mut out) {
-                Ok(v) => v,
-
-                Err(quiche::Error::Done) => {
-                    trace!("done writing");
-                    break;
+        // Handle path events.
+        while let Ok(qe) = conn.poll_path() {
+            match qe {
+                quiche::PathEvent::New(..) => {
+                    panic!("Should not see new path");
```

Comment:
I appreciate this is an semantic error but it would be great to avoid the explicit panic. This makes me wonder, does the library event return such an event?

Perhaps a clearer thing to do would be to use `unreachable!()` ?

Reply:
On a client-side connection, it should not. I replaced that to `unreachable!()`.